modules/update-restricted-signature-digests.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster/restricted-network-update.adoc
+
+:_content-type: PROCEDURE
+[id="update-restricted-signature-digests_{context}"]
+= Retrieving sha256 digests from the release image ConfigMap
+
+In order to update a cluster in a disconnected environment using the `oc adm upgrade` command with the `--to-image` option, you must reference the sha256 sum value that corresponds to your targeted release image.
+This sum value can be acquired by using the `oc get` command to retrieve information about release image signatures on your cluster and their corresponding digests.
+
+.Procedure
+
+. Run the following command on a device that is connected to the internet to get the digest for your target release image:
++
+[source,terminal]
+----
+$ oc adm release info quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE_NUMBER}-${ARCHITECTURE} | sed -n 's/Pull From: .*@//p'
```

Comment:
This approach pulls the release image digest from the canonical image registry for that release image.  The section heading, and references to that section, could probably be agnostic about mechanism with "Retrieving release image digests" or some such, matched to a file name like `update-restricted-image-digest.adoc`.

Reply:
nit: you can skip the `sed` by using JSONPath:

```console
$ oc adm release info -o 'jsonpath={.digest}{"\n"}' quay.io/openshift-release-dev/ocp-release:4.12.14-x86_64
sha256:157cc02d63bfe67988429fd803da632e495e230d811759b1aed1e6ffa7a3f31a
```

Reply:
When we use this approach to set up `oc adm release mirror ...`, we have some lead-in sections explaining how to populate the shell variables:

```console
$ git --no-pager grep -4 'ARCHITECTURE=\|OCP_RELEASE=' modules/update-mirror-repository.adoc
modules/update-mirror-repository.adoc-.. Export the release version:
modules/update-mirror-repository.adoc-+
modules/update-mirror-repository.adoc-[source,terminal]
modules/update-mirror-repository.adoc-----
modules/update-mirror-repository.adoc:$ export OCP_RELEASE=<release_version>
modules/update-mirror-repository.adoc-----
modules/update-mirror-repository.adoc-+
modules/update-mirror-repository.adoc-For `<release_version>`, specify the tag that corresponds to the version of {product-title} to which you want to update, such as `4.5.4`.
modules/update-mirror-repository.adoc-
--
modules/update-mirror-repository.adoc-.. Export the type of architecture for your cluster:
modules/update-mirror-repository.adoc-+
modules/update-mirror-repository.adoc-[source,terminal]
modules/update-mirror-repository.adoc-----
modules/update-mirror-repository.adoc:$ ARCHITECTURE=<cluster_architecture> <1>
modules/update-mirror-repository.adoc-----
modules/update-mirror-repository.adoc-<1> Specify the architecture of the cluster, such as `x86_64`, `aarch64`, `s390x`, or `ppc64le`.
modules/update-mirror-repository.adoc-
modules/update-mirror-repository.adoc-.. Export the path to the directory to host the mirrored images:
```

If feels strange to me to talk through that in one context, but feel like customers can self-serve in another.  I'm not sure if it's worth copy/pasting the setup instructions over here, or putting them in a module that can be consumed in both locations (`update-find-by-tag-release-pullspec.adoc`?).  Or maybe we think folks can self-serve in this context.  I'm agnostic on next-steps  vs. this aspect, but thought I'd point out the divergence in case other folks had opinions.